### PR TITLE
Feed | New icon

### PR DESCRIPTION
**Pull request by justinara**

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**
Commit: **77e0961ab24544dd08148634312546e6f4f5fb0c**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2343/

**Review by milkyfruit — COMMENTED** [review]
@justinara do you think "Activity" is the right name for the icon?
Maybe name "Feed" would reflect better its look and also would match Activity criteria.
What do you think?

**Comment by justinara** [comment]
> 

yes, I think 'Feed' could work quite well. I'll update the name.

**Review by milkyfruit — DISMISSED** [review]

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 1]
```diff
@@ -858,6 +858,13 @@ sizes: { "18": "HideSidebarSmall", "24": "HideSidebar" },
     tags: ["object", "code", "data"],
     sizes: { "18": "CodeObjectSmall", "24": "CodeObject" },
   },
+{
+    title: "Feed",
+    category: "General",
+    description: "Button - displays a recent activity of a page",
+    tags: ["activity", "feed", "notifications"],
+    sizes: { "18": "ActivitySmall", "24": "Activity" },
```
Comment (Zivilekin) [inline packages/wix-ui-icons-common/src/general/metadata.ts:866]:
```suggestion
    sizes: { "18": "FeedSmall", "24": "Feed" },
```

**Review by Zivilekin — APPROVED** [review]

**Pull request merged**